### PR TITLE
[W-12690315] adjusted styles

### PR DESCRIPTION
**Pull request by agansop**

### CODE REVIEW

**Review by cheungaryk — CHANGES_REQUESTED** [review]
some things are not functioning as expected. Please watch my video review: https://www.loom.com/share/8fa9ee977b99465a932902491ad438b5

File: src/css/specific/notice-banner.css [hunk 1]
```diff
@@ -4,10 +4,11 @@
   height: fit-content;
   margin-bottom: -20px;
   margin-top: -12px;
```
Comment (cheungaryk) [inline src/css/specific/notice-banner.css:6]: you can take these margins out. They seem to cause the toolbar and the notice banner to overlap.

Screenshot 2023-03-30 at 3 01 00 PM

**Review by cheungaryk — CHANGES_REQUESTED** [review]

**Review by cheungaryk — CHANGES_REQUESTED** [review]

File: src/css/specific/notice-banner.css (hunk 1 above)
Comment (agansop) [reply src/css/specific/notice-banner.css]: Fixed

File: src/css/specific/notice-banner.css [hunk 2]
```diff
@@ -4,10 +4,11 @@
...
+  position: sticky;
   z-index: var(--z-backdrop);
 
   & p {
-    margin: 12px 40px -20px 40px;
+    margin: 0% 40px 0% 40px;
```
[the remark this answers is not in the document]
Comment (agansop) [reply src/css/specific/notice-banner.css]: Fixed

**Review by cheungaryk — APPROVED** [review]

**Pull request merged**